### ISSUE TITLE
помогите решить проблему ошибки No added CSS/SCSS plugins No added JS plugins

### ISSUE THREAD

**Issue by Sellato** [budfy/Easy-webdev-startpack#19]
Использую node v12.9.1 на более поздних версиях вообще не собирается, куча ошибок :(

На `node v12.9.1` запускается, но есть две ошибки `No added CSS/SCSS plugins` `No added JS plugins`.
Опыта у меня ноль, только начинаю разбираться, поэтому сам устранить проблему не могу...
Подскажите, как это пофиксить.

Спасибо.

![Снимок_экрана_2021-12-02_13-54-44](https://user-images.githubusercontent.com/5456419/144391046-84f85932-0d6e-4142-adb7-c050afc1cdef.png)

**Comment by GlobalAnabiosis** [comment]
> Использую node v12.9.1 на более поздних версиях вообще не собирается, куча ошибок :(
> 
> На `node v12.9.1` запускается, но есть две ошибки `No added CSS/SCSS plugins` `No added JS plugins`. Опыта у меня ноль, только начинаю разбираться, поэтому сам устранить проблему не могу... Подскажите, как это пофиксить.
> 
> Спасибо.
> 
> ![Снимок_экрана_2021-12-02_13-54-44](https://user-images.githubusercontent.com/5456419/144391046-84f85932-0d6e-4142-adb7-c050afc1cdef.png)
> 
> PS
> 
> Попробовал запустить Ваш другой проект https://github.com/budfy/8-march-test на той же платформе все собралось без проблем и работает..

Подскажи, нашёл решение?

**Comment by Sellato** [comment]
нет, забросил, взал некоторые задачи, которые меня заинтересовали. На node 14.13.1 тоже самое.

**Comment by Sellato** [comment]
есть новый релиз, в котором нет этой проблемы, https://github.com/budfy/viktorsur_1
либо, можно тупо потырить от туда две задачи и проинсталировать swiper `npm i swiper`

`libs_js.js` и `libs_style.js`

**Comment by Sellato** [comment]
> Подскажи, нашёл решение?

Перепилил на работу с nodejs 17.x и gulp-sass 5.x [если интересно](https://github.com/Sellato/fork-easy-webdev-startpack-node-v17-gulp-sass-v5)
